exercises/practice/zipper/.meta/config.json

```diff
@@ -1,6 +1,62 @@
 {
   "blurb": "Creating a zipper for a binary tree.",
   "authors": [],
```

Comment:
another empty one, added in https://github.com/exercism/elixir/commit/d0d4cd185bf7e90328af2299f49acda190fd85b4